maxim/logger/pydantic_ai/client.py

```
@@ -0,0 +1,737 @@
+        elif hasattr(response, 'prompt_tokens'):
+            # Direct attributes on response
+            usage_info["prompt_tokens"] = getattr(response, 'prompt_tokens', 0)
+            usage_info["completion_tokens"] = getattr(response, 'completion_tokens', 0)
+            usage_info["total_tokens"] = getattr(response, 'total_tokens', 0)
+    except Exception as e:
+        scribe().debug(f"[MaximSDK] Error extracting usage: {e}")
+    
+    return usage_info
+
+
+def extract_model_info(model_obj) -> dict:
```

Ruff (0.12.2)

Missing type annotation for function argument `model_obj`

(ANN001)